modules/@apostrophecms/page-type/index.js

```diff
@@ -175,6 +175,11 @@ module.exports = {
             // acceptable coverage here for now
             throw self.apos.error('invalid', 'You must unpublish child pages before unpublishing their parent.');
           }
+        },
+        async parkedPageMustNotBeUnpublished(req, published) {
+          if (published.parked) {
+            throw self.apos.error('invalid', 'apostrophe:pageIsParkedAndCannotBeUnpublished');
+          }
```

Comment:
ℹ️ the check of parked pages that should not be removed is actually made in `modules/@apostrophecms/page-type/index.js`:

```js
      beforeDelete: {
        async checkForParked(req, doc, options) {
          if (doc.level === 0) {
            throw self.apos.error('invalid', 'The home page may not be removed.');
          }
          if (doc.parked) {
            throw self.apos.error('invalid', 'This page is "parked" and may not be removed.');
          }
        },
```

But without this additional `parkedPageMustNotBeUnpublished` check, parked pages can still get unpublished:

<img width="704" alt="image" src="https://user-images.githubusercontent.com/8301962/168593620-666718d8-be66-4d88-87c7-5defbd40db3f.png">

That's due to the fact the we edit the draft in the `unpublish` method:

```js
        await self.apos.doc.db.updateOne(
          { _id: draft._id },
          {
            $set: {
              modified: true,
              lastPublishedAt: null
            }
          }
        );
```